bundles/io/org.eclipse.smarthome.io.rest.core/OSGI-INF/loginresource.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2016 by the respective copyright holders.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.smarthome.io.rest.core.auth.LoginResource">

```

Comment:
According to our discussion, auth must be built into the client apps, so I guess we should remove the LoginResource from this PR as it does not really provide any value.